data/tlg0341/tlg002/tlg0341.tlg002.perseus-grc1.xml

```diff
@@ -1,1606 +1,1604 @@
-<l n="1471">ἐτητύμως ἄψορρον ὤτρυνας τρόχιν.</l>
-<l n="1472">δαίμων δέ φήμας εἰς τὸ λῷον ἐκδραμεῖν</l>
-<l n="1473">τεύξειεν, ὅσπερ σῶν προκήδεται θρόνων,</l>
-<l n="1474">σώζων παλαιὰν Βεβρύκων παγκληρίαν.</l>
-</div1>
-</body>
-</text>
-</TEI.2>
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-model href="http://www.stoa.org/epidoc/schema/9.1/tei-epidoc.rng" schematypens="http://relaxng.org/ns/structure/1.0"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:lang="en" xmlns:tei="http://www.tei-c.org/ns/1.0">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="grc">Ἀλεξάνδρα</title>
+                <author>Lycophron</author>
+                <sponsor>Perseus Project, Tufts University</sponsor>
+                <principal>Gregory Crane</principal>
+                <respStmt>
+                    <resp>Prepared under the supervision of</resp>
+                    <name>Lisa Cerrato</name>
+                    <name>William Merrill</name>
+                    <name>Elli Mylonas</name>
+                    <name>David Smith</name>
+                </respStmt>
+                <funder n="org:Google">Google Digital Humanities Awards Program</funder>
+            </titleStmt>
+            <extent />
+            <publicationStmt>
+                <publisher>Trustees of Tufts University</publisher>
+                <pubPlace>Medford, MA</pubPlace>
+                <authority>Perseus Project</authority>
+            </publicationStmt>
+            <sourceDesc>
+                <biblStruct>
+                    <monogr>
+                        <author>Lycophron</author>
+                        <title>Alexandra</title>
+                        <editor>A.W. Mair</editor>
+                        <imprint>
+                            <publisher>London: William Heinemann; New York: G.P. Putnam's Sons</publisher>
+                            <date>1921</date>
+                        </imprint>
+                    </monogr>
+                </biblStruct>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <refsDecl n="CTS">
+                <cRefPattern n="line" matchPattern="(\w+)"
+                    replacementPattern="#xpath(/tei:TEI/tei:text/tei:body/tei:div/tei:l[@n='$1'])">
+                    <p>This pointer pattern extracts the line.</p>
+                </cRefPattern>
+            </refsDecl>
+        </encodingDesc>
+        <profileDesc>
+            <langUsage>
+                <language ident="grc">Greek</language>
+                <language ident="en">English</language>
+            </langUsage>
+        </profileDesc>
+        <revisionDesc>
+            <change when="2023-07-02" who="Charles Pletcher">EpiDoc and CTS Conversion</change>
+        </revisionDesc>
+    </teiHeader>
+    <text n="stoa0045.stoa0">
+        <body>
+            <div type="edition" n="urn:cts:greekLit:tlg0341.tlg002.perseus-grc1" xml:lang="grc">
+                <pb n="494" />
```

Comment:
pb can also be xml:id instead of n (both are used presently, but I have been trying to use the former).
(the space is superfluous in the tag but not a  big deal)